### PR TITLE
#165716231 Fix Reactions Buttons on Scroll 

### PR DESCRIPTION
**Pull request by YvesIraguha**
#### What does this PR do?
- Create styles for fixing reactions buttons on viewing article
#### Description of Task to be completed?
- Adjust styles of view article
- Add reporting form on view article
#### How should this be manually tested?
- Clone the repos and check out the branch `ch-fix-reactions-buttons-on-scroll-165716231`
- Run `yarn` command to install all dependencies
- Run `yarn test` to run all tests. You should see all the tests passing. 
- start your backend server and front-end server by `yarn start`
- Create articles in your backend. 
- Grab a slug of one article. 
- Open your browser and navigate to `/articles/:slug`. You should see the article. When you scroll down, reactions buttons should stay visible. When you click on more button, you should see the report article. When you click on the report article, the report article form should appear. When you click cancel on report article form, the reporting form should disappear. 

#### What are the relevant pivotal tracker stories?

#165716231
#### Relevant screenshot
Screen Shot 2019-05-02 at 16 20 19

### CODE REVIEW

**Review by niomwungeri-fabrice — APPROVED** [review]

File: src/components/reportingForm/ReportingForm.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "../common/Buttons/BasicButton";
+import Input from "../common/Inputs/TextInput";
+
+export const MoreReactions = props => {
```
Comment (abayo-luc) [inline src/components/reportingForm/ReportingForm.js]: you can separate these two components into different files.

**Review by abayo-luc — CHANGES_REQUESTED** [review]
Well done Mr. @YvesIraguha

**Review by abayo-luc — APPROVED** [review]

File: src/__tests__/__actions__/readArticleAction.test.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ import {
 const middlewares = [thunk];
 const mockStore = configureMockStore(middlewares);
 
-const BASE_URL = process.env.API_BASE_URL;
+const BASE_URL = process.env.API_URL;
```
Comment (oesukam) [inline src/__tests__/__actions__/readArticleAction.test.js]: @YvesIraguha Can't you destruct?

File: src/views/ReadArticle.js [hunk 3]
```diff
@@ -63,6 +70,22 @@ export class Article extends Component {
...
+    const { displayModal } = this.state;
+    if (displayModal === true) {
+      return this.setState({ displayModal: false });
+    }
+    return this.setState({ displayModal: true });
+  };
+
+  toggleReportingModal = () => {
+    const { reportingForm } = this.state;
+    if (reportingForm === true) {
```
Comment (oesukam) [inline src/views/ReadArticle.js]: @YvesIraguha Change to `if(reportingForm)`

File: src/views/ReadArticle.js [hunk 4]
```diff
@@ -254,7 +275,7 @@ export class Article extends Component {
                       title="Delete"
                     />
                     <Button
-                      className="btn delete_article"
+                      className="btn edit_article"
                       onClick={() => this.redirectToEdit()}
```
Comment (oesukam) [inline src/views/ReadArticle.js]: @YvesIraguha Should be called by reference

**Review by oesukam — CHANGES_REQUESTED** [review]

**Review by oesukam — APPROVED** [review]

**Pull request merged**